scripts/builtin/lmCG.dml

```diff
@@ -0,0 +1,180 @@
+#-------------------------------------------------------------
+#
+# Copyright 2019 Graz University of Technology
```

Comment:
Use the modifications-license as the original linregCG was copied from Apache SystemML.

Reply:
I am sorry could you explain which comment I would have to add instead. Is it the same that as the one inside LinearRegCG.dml?

Reply:
Found one in DenseBlockDRB, will add it

Reply:
exactly